php-checks/src/test/resources/checks/UnusedFunctionParametersCheck.php

```diff
@@ -303,3 +303,27 @@ public function normalMethod($subject, $arg1, $arg2) // Noncompliant 3
         return;
     }
 }
+
+// Magic methods - methods starting with __ should not raise issues
+class MagicMethodsClass
+{
+    // Destructor - should not raise issues
+    public function __destruct()
```

Comment:
This shouldn't be the only "real" magic method that we test, because it doesn't have any parameters, where we would raise. Please add another example (with something like `__set`, for example), and also a test case for `__construct` with unused parameters.